frontend/test/metabase/scenarios/admin/settings/settings.cy.spec.js

```diff
@@ -356,6 +356,26 @@ describe("scenarios > admin > settings", () => {
       cy.findByText("Sorry, something went wrong. Please try again.");
     });
 
+    it("should send a test email for a valid SMTP configuration", () => {
+      // We must clear maildev inbox before each run - this will be extracted and automated
+      cy.request("DELETE", "http://localhost:80/email/all");
+      cy.request("PUT", "/api/setting", {
+        "email-smtp-host": "localhost",
+        "email-smtp-port": "25",
+        "email-smtp-username": "admin",
+        "email-smtp-password": "admin",
+        "email-smtp-security": "none",
+        "email-from-address": "mailer@metabase.test",
+      });
+      cy.visit("/admin/settings/email");
+      cy.findByText("Send test email").click();
+      cy.findByText("Sent!");
+      cy.request("GET", "http://localhost:80/email").then(({ body }) => {
+        const EMAIL_BODY = body[0].text;
```

Comment:
Nit: I think the naming should be camelCase here because it is not a constant

Reply:
How do you mean it's not a constant? It will always return the same value for this particular email.

Btw, I'm glad you brought it up because we definitely need some "rules" re: code style. There are far too many different styles in our code base.

Reply:
It is not a constant by meaning its value is not defined before the execution time and it is coming from a maildev server.
Yes, it is time to create a style guide for the front end 👍

Reply:
Ah, ok. Thanks for the explanation.